shared/chat/conversation/info-panel/index.js

```diff
@@ -39,11 +39,23 @@ const _SmallTeamInfoPanel = (props: SmallTeamInfoPanelProps) => (
     <Box style={{...globalStyles.flexBoxColumn, alignItems: 'stretch'}}>
       <Participants
         participants={props.participants}
-        onAddParticipant={props.onAddParticipant}
+        onAddParticipant={props.showTeamButton ? null : props.onAddParticipant}
```

Comment:
the various conditionals on showTeamButton here is gross, but I'm hoping we won't have this flag for too long